control/grpc.py

```diff
@@ -118,12 +118,16 @@ def create_subsystem(self, request, context=None):
 
         self.logger.info(
             f"Received request to create subsystem {request.subsystem_nqn}")
+        min_cntlid = self.config.getint_with_default("gateway", "min_cntlid", 1)
+        max_cntlid = self.config.getint_with_default("gateway", "max_cntlid", 65519)
```

Comment:
Do we want to duplicate SPDK defaults here?  Perhaps a better approach would be to keep the current behavior of not passing anything in the RPC if the respective option is not provided?  @PepperJo, any preference?

Reply:
No preference. Maybe we should print a warning that if not specified in a multi GW group multipathing will not work correctly. Alternatively we want to be more verbose we can also add the option to the omap and let the GW check that it doesn't overlap with any others and fail if it does.

Reply:
@PepperJo I was told that the Ceph-adm will take care of managing the ranges between the different gateways so we just need a way for it to pass the values.

Reply:
Ok that makes sense. No need to get complicated then.

Reply:
@PepperJo failing the gateway seems like an overkill. Especially when there is an easy workaround we used here to test two gateways. We just created a connection and the deleted it on one gateway. The result was that now one gateway advanced the controller id counter so now we could create a connection on both gateways and each one got a different controller id.

Reply:
I mean it's certainly not something a user should experience. But if we transfer this responsibility to cephadm I don't have any objections to not checking.